migrations/20240730163630.sql

```diff
@@ -0,0 +1,6 @@
+-- Rename an index from "posts_user_id_key" to "posts_user_id_idx"
+ALTER INDEX "posts_user_id_key" RENAME TO "posts_user_id_idx";
```

Comment:
This is the bug of https://github.com/ariga/atlas/issues/3006

```suggestion
-- Rename an index from "posts_user_id_key" to "posts_user_id_idx"
ALTER INDEX "app"."posts_user_id_key" RENAME TO "posts_user_id_idx";
```